R/system.R

```diff
@@ -1349,7 +1316,8 @@ isConnecitonPoolEnabled <- function(type){
 getListOfTablesWithODBC <- function(conn){
   topLevels <- odbc::odbcListObjects(conn)
   schemas <- NULL
-  check <- topLevels %>% dplyr::distinct(type) == c("catalog")
+  df <- topLevels %>% dplyr::distinct(type)
+  check <- df$type == c("catalog")
```

Comment:
This will return vector of logical, not a single logical. You will need to add something like any() or all() to make it a single logical.

Reply:
Thank you for checking. Applied `all()` to make it a single logical.